pubtools/_pulp/tasks/delete/base.py

```diff
@@ -0,0 +1,124 @@
+from ast import Del
```

Comment:
This must be a mistake, ast module is for parsing Python.

Reply:
Yes, removed.